packages/api/tests/task.test.js

```diff
@@ -1949,11 +1949,13 @@ describe('Task tests', () => {
           }),
         );
 
+        const { animals, animal_batches, ...patchTaskData } = createdTask;
```

Comment:
`createdTask` (Post request's response body) now includes animals and animal_batches that shouldn't be included in the patch request. This is the adjustment.